content/en/developers/dogstatsd/high_throughput.md

```diff
@@ -218,6 +224,85 @@ Then set the Agent `dogstatsd_so_rcvbuf` configuration option to the same number
 dogstatsd_so_rcvbuf: 26214400
 ```
 
+### Over UDS (Unix Domain Socket)
+
+#### Linux
+
+For UDS sockets, Linux is internally buffering datagrams in a queue if the
+reader is slower than the writer. The size of this queue represents the maximum number
+of datagrams that Linux will buffer per socket. This value can be queried with the following command:
+
+```bash
+sysctl net.unix.max_dgram_qlen
+```
+
+If the value is < 512, you can increase it to 512 or more using this command:
+
+```bash
+sysctl -w net.unix.max_dgram_qlen=512
+```
+
+Add the following configuration to `/etc/sysctl.conf` to make this change permanent:
+
+```conf
+net.unix.max_dgram_qlen = 512
+```
+
+In the same manner, the `net.core.wmem_max` could be incremented to 4MiB to improve
+client writing performances:
+
+```conf
+net.core.wmem_max = 4194304
+```
+
+Then set the Agent `dogstatsd_so_rcvbuf` configuration option to the same number in `datadog.yaml`:
+
+```yaml
+dogstatsd_so_rcvbuf: 4194304
+```
+
+Please note that setting these values will need a reboot of the host.
+
+### Ensure proper packet sizes
+
+Sending packets with an adequate size to Dogstatsd will avoid extra work to the server.
+
+If the packets sent are too small, Dogstatsd will have to pack several together in order
+to process them in batches later in the pipeline. The official Dogstatsd clients are
+capable of grouping metrics to have the best ratio of the number of metrics per packet.
```

Comment:
Since this section is basically only relevant for readers who do not use the official clients, I think it's worth highlighting this first. Something like this as the second sentence of this section could work for example:

```
The latest versions of the official Dogstatsd clients send packets with a size optimized for performance. If you're using them, skip this section, otherwise read on.
```

(not super important, can be addressed separately)